src/components/productDetail.js

```diff
@@ -11,6 +12,15 @@ function ProductDetail({ productInformation }) {
   const buttonText = productInformation.ableToAddCard
     ? "Agregar al carrito"
     : "En carrito";
+
+    const renderStars = () => {
+      let stars = [];
+      for (let i = 1; i <= 5; i++) {
+        stars.push(<Star key={i} filled={i <= productInformation.ratingAverage} />);
```

Comment:
Let's try to keep this login on the return, in other words, we have to iterate some list by using map function and then return the Star component you created.
You could create a list from a number like [...Array(5)] and use map to render the stars, where on the map callback function you get the logic to know when it should be filled